commands/FBDebugCommands.py

```diff
@@ -184,3 +185,27 @@ def description(self):
 
   def run(self, arguments, options):
     fb.evaluateEffect('[[UIApplication sharedApplication] performSelector:@selector(_performMemoryWarning)]')
+
+class FBDebuggingInformationOverlayCommand(fb.FBCommand):
+  def name(self):
+    return 'dio'
```

Comment:
Did you have any alternative names you considered? I think `dio` is good as an alias for a more memorable name, but I'm not sure what to call that. `debugoverlay`?

Reply:
How about `doverlay`? Just like the command `dcomponents` which is already shipped with Chisel :)

Reply:
`doverlay` 👍